ci/docker/ml.build.Dockerfile

```diff
@@ -34,7 +34,7 @@ fi
 if [[ "$RAYCI_LIGHTNING_2" == "true" ]]; then
   pip uninstall -y pytorch-lightning
   # todo move to requirements-test.txt
-  pip install lightning==2.0.4 pytorch-lightning==2.0.4  
```

Comment:
We can revert this and leave the test as flaky to unblock this PR and handle it separately.